content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -185,13 +185,14 @@ for i in range(1, EPOCHS+1):
 The line `torch.save(model.state_dict(), "model.pt")` updates the checkpoint
 file.
 
-The `dvclive.log(k, v)` line stores the metric _k_ with a value _v_ in plain
-text files in the _dvclive_ directory by default.
-
-The `dvclive.next_step()` line tells DVC that it can take a snapshot of the
-entire workspace and version it with Git. It's important that with this approach
-only code with metadata is versioned in Git (as an ephemeral commit), while the
-actual model weight file will be stored in the DVC data cache.
+You can read about what the line `dvclive.log(k, v)` does in the
+[`dvclive.log`](/doc/dvclive/api-reference/log) reference.
```

Comment:
```suggestion
[`dvclive.log()`](/doc/dvclive/api-reference/log) reference.
```

Reply:
Probably other links in all the files are missing `()` too.

Reply:
Copy that. I wasn't sure about whether to use `()` in the links or not